src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -313,7 +313,7 @@ export class ResultsViewPageStore {
                 this._oncoKb = val;
             },
             get oncoKb() {
-                return this._oncoKb && !store.didOncoKbFailInOncoprint;
+                return AppConfig.serverConfig.show_oncokb && this._oncoKb && !store.didOncoKbFailInOncoprint;
```

Comment:
what that didOncoKbFail thing?

Reply:
checking if oncokb promise has failed, but I cant use isError because of the way we did it with returning an Error object as result so that the whole app doesnt fail via error result causing all the dependent promises to error.